src/funding-address.ts

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+import type {
+  Marqeta,
+  MarqetaOptions,
+} from './'
+
+export interface FundingAddress {
+  userToken?:	string;
+  businessToken?:	string;
+  token?: string;
+  firstName:	string;
+  lastName:	string;
+  address1:	string;
+  address2?:	string;
+  city:	string;
+  state:	string;
+  zip?:	string;
+  country:	string;
+  phone?:	string;
+  isDefaultAddress?:	boolean;
+  active?:	boolean;
+  postalCode?: string;
+}
```

Comment:
the indentation of this is really odd... not at all like the other interfaces you have... why the difference?